_sass/sidebar.scss

```diff
@@ -1,10 +1,53 @@
+$link-left-padding: 0.5em;
+$indent-space: 1em;
+$vertical-line-offset: -0.3em;
+
 .sidebar {
   ul {
-    margin-left: 1em;
-
-    li {
-      font-size: $foxtrot;
-      list-style: square inside;
+	  position: relative;
+	  left: 10px;
+	  padding-left: 5px;
+	  border-left: solid 1px #cccccc;
+  }
+  li {
+    font-size: $foxtrot;
+    list-style-type: none;
+    padding-right: 8px;
+    a {
+      display: block;
+      border-radius: 4px;
+      &:hover {
+        background-color: #0000FF10;
+      }
+      &::before {
+        content: "";
+        padding-left: $link-left-padding;
+      }
+      text-decoration: none;
+    }
+  }
+  // Category
+  & > li {
+    & > a {
```

Comment:
Minor: maybe combine this selector onto a single line? (`& > li > a {` rather than nesting a single child)